onedal/__init__.py

```diff
@@ -32,4 +32,4 @@
     import onedal._onedal_py_host as _backend
     _is_dpc_backend = False
 
-__all__ = ['primitives', 'svm']
+__all__ = ['primitives', 'svm', 'neighbors']
```

Comment:
Sort it in alphabetical order

Reply:
Follow this rule, please, in the other parts of the code.

Reply:
applied